test/assignment5/sockettest.sh

```diff
@@ -81,7 +115,19 @@ if [ -e long_string.txt ]; then
     sendstring=`cat long_string.txt`
     test_send_socket_string ${sendstring} ${comparefile}
 fi
+
+if [ -e multiple_packets.txt ]; then
+    echo "Sending Multiple packets from multiple_packets.txt file"
+
+    sendstring=`cat multiple_packets.txt`
+
+	test_send_multiple_packets "${sendstring}" "${comparefile}"
```

Comment:
@DeekshithPatil probably have a rogue /t tab character here.  Try deleting to the beginning of the line and replacing with spaces instead of tabs.